src/components/Header/Header.module.scss

```diff
@@ -65,6 +65,7 @@
     .app_bar {
       background-image: url('./gc_seal_top_64.png');
       height: $header-height-sm;
+      background-color: var(--mui-palette-primary-main);
```

Comment:
I think you missed the `$neutral-white` at the bottom of this component.

Reply:
Yeah, I haven't gone through header fully yet(just with my initial experimentation, but this will still work for now, will be changed when we get to that component.